protocols/raft/src/test/java/io/atomix/protocols/raft/zeebe/partition/ZeebeRaftPartitionGroup.java

```diff
@@ -1,78 +0,0 @@
-/*
- * Copyright 2016-present Open Networking Foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.atomix.protocols.raft.zeebe.partition;
-
-import io.atomix.primitive.partition.ManagedPartitionGroup;
-import io.atomix.primitive.partition.PartitionGroup;
-import io.atomix.protocols.raft.partition.RaftPartitionGroup;
-import io.atomix.protocols.raft.partition.RaftPartitionGroupConfig;
-import io.atomix.utils.serializer.Namespace;
-import io.atomix.utils.serializer.Namespaces;
-
-public class ZeebeRaftPartitionGroup extends RaftPartitionGroup {
-
-  public static final ZeebeRaftPartitionGroup.Type TYPE = new ZeebeRaftPartitionGroup.Type();
-
-  public ZeebeRaftPartitionGroup(RaftPartitionGroupConfig config) {
-    super(config);
-  }
-
-  /**
```

Comment:
We dont need this anymore?

Reply:
No, this was only necessary because I wasn't sure what the consequences of having a transient field on the partition group config would do. If we always define all groups on every node, then it's safe. If we didn't then we'd need to resort to this trick again of having custom partition group extensions.

Reply:
ok thanks